### PR TITLE
feat/#52 observe and update the input

### LINKED ISSUE #52: Observe And Update The Input
### Explanation:
Insert ticket view, change styles in header, add back button, fonts, text field and button styles, design attached link in figma to download missing resources.

### SCREESHOTS

view_start

### CHECKLIST

- [ ] Change the background color of the header with rounded corners to the layout color `#9DBDFD`
- [ ] Place back button aligned to the top left according to design
- [ ]Change title and subtitle text styles in header type: regular and bold popups to appropriate color `#002740`
- [ ] Change title styles in popins bold uppercase font color `#005D9D`
- [ ] Change text field size and color, background color `#C4C4C4` to 23%, border `#5F6769` to 1px, font uses regular popins, all fields to uppercase.
- [ ] Change button color color `#005D9D` appears in medium caps.

### REFERENCES
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=407%3A46
https://www.figma.com/file/tbhMgDT10bbVjB9GazF97y/Dasher?node-id=619%3A102

### PR DESCRIPTION
**Pull request by arodriguezl9302**
#### ATTENTION

PLEASE NOTE THAT TO AVOID PROBLEMS RUNNING THIS PR, YOU NEED TO HAVE PR#50 RUNNING

### Description

Changed header styles, added back button, fonts, text field and button styles

fixes #52 

#### Screenshots

image

#### Type of change

New feature (non-breaking change which adds functionality)

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

1. Go to '/check'

### Checklist:

- [x] Change header background color with rounded corners according to design color #9DBDFD
- [x] Place back button aligned to the top left according to design
- [x] Change title and subtitle text styles in header type: regular and bold popins as appropriate color #002740
- [x] Change title styles in bold uppercase popins font color #005D9D
- [x] Change size and color of text field, background color #C4C4C4 23%, border #5F6769 to 1px, font used regular popins, all fields capitalized.
- [x] Change button color color #005D9D popins medium uppercase font.

### CODE REVIEW

**Comment by heliomar-pena** [comment]
I will wait to #70 was merged to check this, to avoid conflicts

**Review by e-azocar — APPROVED** [review]

**Comment by angelik0828** [comment]
It looks good, I have some suggestions, the button is very small, it should be almost the size of the text field, the ticket image does not go in the layout, you need to change or remove it, please check the margins. that there in the word "hello" at the bottom, attached image with suggestion.

image

image

File: src/components/Input.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const Input = styled.input`
     ({bgColor}) =>
       bgColor === 'gray' &&
       css`
-        background-color: var(--light-gray);
+       //background-color: var(--light-gray);
```
Comment (heliomar-pena) [inline src/components/Input.jsx]: Please remove comment

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Please remove comments

**Comment by arodriguezl9302** [comment]
@Katsu08 @angelik0828

I changed the format of the title and subtitle, removed the notes icon and made the header a bit bigger 

image

File: src/pages/Chat/Chat.styled.jsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ export const HeaderChat = styled.div`
 
 export const HeaderChatImgContainer = styled.div`
   text-align: right;
+  margin-left: -80px;
```
Comment (heliomar-pena) [inline src/pages/Chat/Chat.styled.jsx]: Please do not use negative margin-left with so big numbers, just use it for little adjustment of fewer than ten pixels. Please check here how the image is positioned correctly without using margin-left: 80px https://github.com/Azordev/maps-project/pull/72/commits/9304213eb9507aaf997f3441eab11a4b94ac0fb8

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Comment by angelik0828** [comment]
Thanks for the changes, I see that the button is still in a smaller size, we have talked and it will be handled with another problem since the component used is on another screen.

**Review by angelik0828 — APPROVED** [review]

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Review by Israel-Laguan — APPROVED** [review]

File: src/pages/Check/Check.styled.jsx [hunk 3]
```diff
@@ -2,18 +2,27 @@ import styled from '@emotion/styled';
...
     margin-bottom: 0.3rem;
   }
 `
+
+export const BtnBackContainer = styled.div`
+  margin-top: 10px;
+`
+
+export const SubtitleContainer = styled.div`
+  margin-top: -40px;
```
Comment (heliomar-pena) [inline src/pages/Check/Check.styled.jsx]: Can you check if exists a better solution to position subtitle container?

**Review by heliomar-pena — CHANGES_REQUESTED** [review]
Good job!. I have left you a comment about use of negative margin

**Review by heliomar-pena — APPROVED** [review]
Good work man!

**Pull request merged**